frontend/simple/views/components/Menu/MenuItem.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <!--
+    MenuItem is a wrapper around ListItem with extra styles
+    and to update MenuSelect when an option was selected.
+
+    QUESTION: Is v-bind="$attrs" the correct way to pass down
+    the props and slot from MenuItem to ListItem?
```

Comment:
I don't know off-hand, I see this is [mentioned in the docs](https://vuejs.org/v2/api/#vm-attrs), but I would need to spend a lot more time researching this to be able to give you a good answer. So while my attention is tied up elsewhere perhaps @hubudibu might be able to help here?

Reply:
Yes, I saw it [on the bind part]() so I felt comfortable of doing it this way.

Reply:
yes, it's a totally correct way. I would however prefer to unbundle these attrs and pass them down one-by-one, to make it possible to do type-checks on them, and to make it more explicit reading the code what props this component needs. but that's probably unnecessary at a simple wrapper component like this. it's the usual compactness vs self-documenting readability question :)

Reply:
This is exactly to give totally freedom to the developer to pass whatever props (s)he needs at the moment, it's almost like and HOC (in react vocabulary)